storage/src/main/java/tech/pegasys/teku/storage/server/Database.java

```diff
@@ -72,16 +73,17 @@ void storeFinalizedBlocks(
    * of pruneLimit is to softly cap DB operation time.
    *
    * @param lastSlotToPrune inclusive, not reached if limit happens first
-   * @param pruneLimit soft BlobSidecars (not slots) limit
+   * @param pruneLimit maximum number of slots to prune.
```

Comment:
since you are changing the semantic of this limit, did you checked if we need to update our default value and relative CLI option?

Reply:
Well spotted. Updated to 12 as discussed.